raiden/network/rpc/client.py

```diff
@@ -309,7 +309,7 @@ def _gasprice(self) -> int:
         if self.given_gas_price:
             return self.given_gas_price
 
-        return self.web3.eth.gasPrice
+        return round(TESTNET_GASPRICE_MULTIPLIER * self.web3.eth.gasPrice)
```

Comment:
Defining the constant means that this applies to all networks we use Raiden with. Or, that we have to manually remove this multiplier as soon as we want to release mainnet. 
I think @LefterisJP wanted to add a `--gas-price-multiplier` that can be used while running on testnet. That is, if i understood his comment correctly.

Reply:
As I understood we decided for the easy solution now and will fix this properly for the mainnet release. @LefterisJP

Reply:
Hey @rakanalh and Paul. What I meant was exactly what Paul did. To have a constant to temporarily bump gas price for ropsten testing.

We don't need a new argument since this hack should not make it to the mainnet. And if someone really wants to provide a custom gas price either during testnet release or mainnet release they can so so by providing a specific gas price through the [--gas-price](https://github.com/raiden-network/raiden/blob/cbd6d3edc5684b3f637af7ed1d9e34dcf93e4efc/raiden/ui/cli.py#L373) argument